x-pack/plugins/maps/public/classes/styles/vector/components/legend/marker_size_legend.tsx

```diff
@@ -22,10 +23,13 @@ interface Props {
 
 interface State {
   label: string;
+  maxLabelWidth?: number;
 }
 
 export class MarkerSizeLegend extends Component<Props, State> {
   private _isMounted: boolean = false;
+  private _textRefs: Record<string, RefObject<TextElement>> = {};
```

Comment:
I think this is the correct type.

```suggestion
  private _textRefs: Record<string, RefObject<SVGTextElement>> = {};
```

Reply:
Thanks, that makes sense. Replaced at 4320b8b